explorer/models/azureRegistryNodes.ts

```diff
@@ -202,23 +196,26 @@ export class AzureRepositoryNode extends NodeBase {
             // tslint:disable-next-line:prefer-for-of // Grandfathered in
             for (let i = 0; i < tags.length; i++) {
                 pool.addTask(async () => {
-                    let data = await request.get('https://' + element.repository + '/v2/' + element.label + `/manifests/${tags[i]}`, {
-                        auth: {
-                            bearer: accessTokenARC
-                        }
-                    });
-
-                    //Acquires each image's manifest to acquire build time.
-                    let manifest = JSON.parse(data);
-                    node = new AzureImageNode(`${element.label}:${tags[i]}`, 'azureImageNode');
-                    node.azureAccount = element.azureAccount;
-                    node.password = element.password;
-                    node.registry = element.registry;
-                    node.serverUrl = element.repository;
-                    node.subscription = element.subscription;
-                    node.userName = element.userName;
-                    node.created = moment(new Date(JSON.parse(manifest.history[0].v1Compatibility).created)).fromNow();
-                    imageNodes.push(node);
+                    try {
+                        let data: string = await request.get('https://' + element.repository + '/v2/' + element.label + `/manifests/${tags[i]}`, {
+                            auth: {
+                                bearer: accessTokenARC
+                            }
+                        });
+
+                        //Acquires each image's manifest to acquire build time.
+                        let manifest = JSON.parse(data);
+                        node = new AzureImageNode(`${element.label}:${tags[i]}`, 'azureImageNode');
+                        node.azureAccount = element.azureAccount;
+                        node.registry = element.registry;
+                        node.serverUrl = element.repository;
+                        node.subscription = element.subscription;
+                        node.created = moment(new Date(JSON.parse(manifest.history[0].v1Compatibility).created)).fromNow();
+                        imageNodes.push(node);
+
+                    } catch (error) {
+                        vscode.window.showErrorMessage(error);
```

Comment:
I'm trying to understand where you expect the error to be coming from. Is it from the parse, or the request? I don't see lines 209-213 causing an error. 

If you're surer of where the error is expected to come from, it might be better to wrap just that line in a catch.

Reply:
I anticipate  it would come from the request , that seems reasonable, I think I just moved it to the try to avoid having an additional check in case it failed but it is more unreadable.